packages/xrpl/test/integration/requests/subscribe.test.ts

```diff
@@ -8,27 +7,32 @@ import {
   SubscribeResponse,
 } from 'xrpl-local'
 import { StreamType } from 'xrpl-local/models/common'
+import type { LedgerStreamResponse } from 'xrpl-local/models/methods/subscribe'
```

Comment:
Shouldn't `xrpl-local` be removed here? Or will that happen in a followup?

Reply:
Removed it everywhere I can in tests